modules/seqexec/server/src/main/scala/seqexec/server/gsaoi/GsaoiController.scala

```diff
@@ -32,8 +32,8 @@ trait GsaoiController[F[_]] {
 
   def calcTotalExposureTime(cfg: DCConfig)(
     implicit ev:                 Applicative[F]): F[Time] = {
-    val readFactor  = 2.2
-    val readOutTime = 300
+    val readFactor  = 1.2
+    val readOutTime = 15
```

Comment:
Previous values were taken from the timeout calculation, and were a bit too big for total time estimation.